src/libs/actions/IOU.ts

```diff
@@ -12657,15 +12657,15 @@ function initSplitExpense(transaction: OnyxEntry<OnyxTypes.Transaction>, isOpenC
             splitExpenses: [
                 {
                     transactionID: NumberUtils.rand64(),
-                    amount: Math.floor(transactionDetailsAmount / 2),
+                    amount: calculateIOUAmount(1, transactionDetailsAmount, transactionDetails?.currency ?? CONST.CURRENCY.USD, false),
                     description: transactionDetails?.comment,
                     category: transactionDetails?.category,
                     tags: transaction?.tag ? [transaction?.tag] : [],
                     created: transactionDetails?.created ?? defaultCreated,
                 },
                 {
                     transactionID: NumberUtils.rand64(),
-                    amount: Math.ceil(transactionDetailsAmount / 2),
+                    amount: calculateIOUAmount(1, transactionDetailsAmount, transactionDetails?.currency ?? CONST.CURRENCY.USD, true),
```

Comment:
```suggestion
                    amount: calculateIOUAmount(1, transactionDetailsAmount, transactionDetails?.currency ?? '', true),
```

Reply:
Since we already use the default currency in `calculateAmount` - `getCurrencyUnit`